src/test/java/com/facebook/openwifirrm/optimizers/TestUtils.java

```diff
@@ -463,20 +472,26 @@ private static State.Unit createStateUnit() {
 	 * @param channelWidths array of channel widths (MHz)
 	 * @param txPowers array of tx powers (dBm)
 	 * @param bssids array of BSSIDs
+	 * @param stations 2-D array of client station codes
 	 * @param clientRssis 2-D array of client RSSIs
+	 * @param localtime
 	 * @return the state of an AP with radios described by the given parameters
 	 */
 	public static State createState(
 		int[] channels,
 		int[] channelWidths,
 		int[] txPowers,
 		String[] bssids,
-		int[][] clientRssis
+		String[][] stations,
+		int[][] clientRssis,
+		long localtime
 	) {
 		if (
 			!(channels.length == channelWidths.length &&
 				channelWidths.length == txPowers.length &&
-				txPowers.length == bssids.length)
+				txPowers.length == bssids.length &&
+				channels.length == stations.length &&
```

Comment:
just personal preference, I feel like to keep consistent with the rest of the structure, `bssids.length == stations.length` is better, but up to you